### DISCUSSION TITLE
如何给标绘的点、线、面、配置和显示label文字

### DISCUSSION THREAD

**Discussion by webdq** [zouyaoji/vue-cesium#208] [Q&A]
如何给标绘的点、线、面、配置和显示label文字

**Comment by zouyaoji** [comment]
这个和 https://github.com/zouyaoji/vue-cesium/discussions/206 思路是一样的哈。

![M9~7}I}CJMR7L1Q43~VY9GC](https://user-images.githubusercontent.com/24430116/168951981-9eaae5e0-481e-434a-b278-fb0ac13ef777.jpg)

**Reply by zouyaoji** [reply]
不好意思，我说错了，标绘不行哦。标绘只有pin那个可以配置，目前。

**Reply by zouyaoji** [reply]
有需求的话可以开出来。

**Reply by zouyaoji** [reply]
下个版本将开出来！
